Maddori.Apple/Maddori.Apple/Global/Literal/TextLiteral.swift

```diff
@@ -139,6 +139,7 @@ enum TextLiteral {
     // MARK: - SelectReflectionMemberViewController
     
     static let selectReflectionMemberViewControllerTitleLabel = "각 팀원이 받은 피드백을\n공유해 보세요"
+    static let selectReflectionMemberViewControllerSubtitleLabel = "모든 팀원 버튼을 누르면 회고를 종료할 수 있습니다"
```

Comment:
p1;
```suggestion
    static let selectReflectionMemberViewControllerSubtitleLabel = "모든 팀원을 누르면 회고를 종료할 수 있습니다"
```
피그마에서는 '모든 팀원을' 이라고 되어 있는 것 같아용!

Reply:
확인 !!

[f845aca](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/357/commits/f845aca08b118901aa5180d13c966837cad0e803)

Reply:
예리하다....👍